axon-server-connector/src/test/java/org/axonframework/axonserver/connector/heartbeat/HeartbeatConnectionCheckTest.java

```diff
@@ -0,0 +1,68 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import org.junit.*;
+
+import java.time.Instant;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static java.time.temporal.ChronoUnit.MILLIS;
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.junit.Assert.*;
+
+/**
+ * @author Sara Pellegrini
+ * @since 4.2
```

Comment:
Nit: I believe we do not use the `since` tag for test classes. Not sure what convention would be best in this respect.

Reply:
I've removed the tag